<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>FrameEntreeData</class>
 <widget class="QFrame" name="FrameEntreeData">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>484</width>
    <height>404</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Frame</string>
  </property>
  <property name="frameShape">
   <enum>QFrame::StyledPanel</enum>
  </property>
  <property name="frameShadow">
   <enum>QFrame::Raised</enum>
  </property>
  <widget class="QWidget" name="verticalLayoutWidget">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>10</y>
     <width>361</width>
     <height>271</height>
    </rect>
   </property>
   <layout class="QVBoxLayout" name="MainVLayout">
    <item>
     <widget class="QGroupBox" name="groupBoxDates">
      <property name="title">
       <string>GroupBox</string>
      </property>
      <widget class="QWidget" name="horizontalLayoutWidget">
       <property name="geometry">
        <rect>
         <x>40</x>
         <y>50</y>
         <width>180</width>
         <height>80</height>
        </rect>
       </property>
       <layout class="QHBoxLayout" name="DatesLayout">
        <item>
         <widget class="QDateEdit" name="dateDebut">
          <property name="calendarPopup">
           <bool>true</bool>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QDateEdit" name="dateFin">
          <property name="calendarPopup">
           <bool>true</bool>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
     </widget>
    </item>
    <item>
     <widget class="QGroupBox" name="groupBoxPers">
      <property name="title">
       <string>GroupBox</string>
      </property>
      <widget class="QWidget" name="horizontalLayoutWidget_2">
       <property name="geometry">
        <rect>
         <x>30</x>
         <y>30</y>
         <width>422</width>
         <height>196</height>
        </rect>
       </property>
       <layout class="QHBoxLayout" name="GroupePersLayout"/>
      </widget>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
